src/mokka/src/main/java/pl/hycom/mokka/endpoint/FileEndpoint.java

```diff
@@ -0,0 +1,90 @@
+package pl.hycom.mokka.endpoint;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.hycom.mokka.service.file.FileService;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.net.URLConnection;
+import java.util.Objects;
+
+/**
+ * Rest controller responsible for file management
+ *
+ * @author Mariusz Krysztofowicz (mariusz.krysztofowicz@hycom.pl)
+ */
+@RequestMapping(value = "/files")
+@RestController
+public class FileEndpoint {
+    private static final Logger LOG = Logger.getLogger(FileEndpoint.class);
```

Comment:
Please do not point to logger specific implementation. Would be great to keep with abstract SLF4J imports